profiles/Im-vishalanand.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/108060013/211681340-3562af47-ad3f-4b9e-82b0-ba6b120ab123.png",
+    "issueId": 2701,
+    "name": "Kumar Vishal Anand",
+    "username": "Im-vishalanand"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```